crates/chia-consensus/src/gen/conditions.rs

```diff
@@ -5636,3 +5655,59 @@ fn test_message_eligible_for_ff() {
         assert_eq!((cond.spends[1].flags & ELIGIBLE_FOR_FF), 0);
     }
 }
+
+#[cfg(test)]
+enum DedupCoinOutput {
+    Excess,
+    Equal,
+    Absorb,
+}
+
+#[cfg(test)]
+#[rstest]
+fn test_eligible_for_dedup(
+    #[values(
+        DedupCoinOutput::Excess,
+        DedupCoinOutput::Equal,
+        DedupCoinOutput::Absorb
+    )]
+    output: DedupCoinOutput,
+) {
+    let coin1_output = match output {
+        DedupCoinOutput::Excess => 122,
+        DedupCoinOutput::Equal => 123,
+        DedupCoinOutput::Absorb => 124,
+    };
+
+    let coin2_output = match output {
+        DedupCoinOutput::Excess => 124,
+        DedupCoinOutput::Equal => 123,
+        DedupCoinOutput::Absorb => 122,
+    };
```

Comment:
It's not obvious to me why you can't just hard-code the coin created by this spend, or just not have it create any coins at all. It's spend 0 that's being tested, right? And the only reason to have a second coin is to avoid the `MINTING_COIN` error, as I understand it.